tests/utils/test_dag_processing.py

```diff
@@ -149,11 +181,13 @@ def test_find_zombies(self):
             ti = TI(task, DEFAULT_DATE, State.RUNNING)
             local_job = LJ(ti)
             local_job.state = State.SHUTDOWN
-            local_job.id = 1
```

Comment:
job.id is an auto-inc primary key. Because of the new test I added, this was failing some of the time. So the change below uses the auto-inc to get the value.

As the comment I added said, if we had a proper relation then we could do 

```python
ti.job = local_job
```

And then a single commit would work and SQLA would handle this for us under the hood.

Reply:
(moved this to it's own PR.)